aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -263,17 +264,38 @@ module AptosFramework::Account {
     }
 
     fun multi_agent_script_prologue(
-        _sender: signer,
-        _txn_sequence_number: u64,
-        _txn_sender_public_key: vector<u8>,
-        _secondary_signer_addresses: vector<address>,
-        _secondary_signer_public_key_hashes: vector<vector<u8>>,
-        _txn_gas_price: u64,
-        _txn_max_gas_units: u64,
-        _txn_expiration_time: u64,
-        _chain_id: u8,
-    ) {
-        assert!(false, Errors::invalid_argument(EMULTI_AGENT_NOT_SUPPORTED));
+        sender: signer,
+        txn_sequence_number: u64,
+        txn_sender_public_key: vector<u8>,
+        secondary_signer_addresses: vector<address>,
+        secondary_signer_public_key_hashes: vector<vector<u8>>,
```

Comment:
it's very interesting that we do the hash inside move for primary but not for secondary lol

Reply:
yes... I hadn't noticed and then had to double back to fix my code 😭 

we should probably unify this awkwardness...

I also wonder if there is some way we can offer this cool function nonsense to other apps.